src/services/api/AxiosInstance.ts

```diff
@@ -11,6 +10,15 @@ import tracer from "@utils/tracer"
 import { customHeaderInterpreter } from "@root/utils/headerInterpreter"
 import { tokenServiceInstance } from "@services/db/TokenService"
 
+import { statsService } from "../infra/StatsService"
+
+const GITHUB_EXPERIMENTAL_TRIAL_SITE = "pa-corp"
```

Comment:
Is this possible to be an array? if we put as an array of sites, we can still track on per site level metric right, i.e. filter githubApiCalls made by sites?

Reply:
yes